contracts/riskon/UniswapV3RiskOnVault.sol

```diff
@@ -0,0 +1,673 @@
+                _claimAmounts[0] -= claimAmount0Fee;
+                IERC20Upgradeable(token0()).safeApprove(address(treasury), 0);
+                IERC20Upgradeable(token0()).safeApprove(address(treasury), claimAmount0Fee);
+                treasury.receiveProfitFromVault(token0(), claimAmount0Fee);
+            }
+            if (_claimAmounts[1] > 0) {
+                uint256 claimAmount1Fee = _claimAmounts[1] * profitFeeBps / 10000;
+                _claimAmounts[1] -= claimAmount1Fee;
+                IERC20Upgradeable(token1()).safeApprove(address(treasury), 0);
+                IERC20Upgradeable(token1()).safeApprove(address(treasury), claimAmount1Fee);
+                treasury.receiveProfitFromVault(token1(), claimAmount1Fee);
+            }
+        }
+        emit StrategyReported(_rewardsTokens, _claimAmounts);
+    }
+
+    /// @notice Lend
+    /// @param _amount The amount of lend
+    function lend(uint256 _amount) external isOwner whenNotEmergency nonReentrant override {
+        IERC20Upgradeable(wantToken).safeTransferFrom(msg.sender, address(this), _amount);
+
+        if (manageFeeBps > 0 && address(treasury) != address(0)) {
+            uint256 manageFee = _amount * manageFeeBps / 10000;
+            IERC20Upgradeable(wantToken).safeApprove(address(treasury), 0);
+            IERC20Upgradeable(wantToken).safeApprove(address(treasury), manageFee);
+            treasury.receiveManageFeeFromVault(wantToken, manageFee);
+            _amount -= manageFee;
+        }
+
+        __addCollateral(_amount.mul(2).div(3));
+        __borrow(uniswapV3RiskOnHelper.calcCanonicalAssetValue(wantToken, _amount.div(3), borrowToken));
+        (, int24 _tick,,,,,) = pool.slot0();
+        if (baseMintInfo.tokenId == 0) {
+            depositTo3rdPool(_tick);
+        } else {
+            if (shouldRebalance(_tick)) {
+                rebalance(_tick);
+            } else {
+                // addLiquidityTo3rdPool
+                uint256 _balance0 = balanceOfToken(token0());
+                uint256 _balance1 = balanceOfToken(token1());
+                if (_balance0 > 0 && _balance1 > 0) {
+                    //add liquidity
+                    nonfungiblePositionManager.increaseLiquidity(INonfungiblePositionManager.IncreaseLiquidityParams({
+                    tokenId : baseMintInfo.tokenId,
+                    amount0Desired : _balance0,
+                    amount1Desired : _balance1,
+                    amount0Min : 0,
+                    amount1Min : 0,
+                    deadline : block.timestamp
+                    }));
+                    _balance0 = balanceOfToken(token0());
+                    _balance1 = balanceOfToken(token1());
+                }
+                if (_balance0 > 0 && _balance1 > 0) {
+                    //add liquidity
+                    nonfungiblePositionManager.increaseLiquidity(INonfungiblePositionManager.IncreaseLiquidityParams({
+                    tokenId : limitMintInfo.tokenId,
```

Comment:
Can limit order add two tokens into LP V3 NFT ?

Reply:
&& to ||